src/mmw/js/src/core/chart.js

```diff
@@ -380,8 +380,202 @@ function renderCompareMultibarChart(chartEl, name, label, colors, stacked, yMax,
+                                    "opacity": 0,
+                                    "fill-opacity": 0,
+                                    "filter": "url(#drop-shadow)"
+                                });
+                            });
+
+                        // add background shape
+                        parentG.append("rect")
+                            .each(function() {
+                                d3.select(this).attr({
+                                    "transform": b.attr("transform"),
+                                    "y": (parseFloat(b.attr("y")) + bgVertOffset),
+                                    "x": (parseFloat(b.attr("x")) + (parseFloat(barWidth) / 2) - (bgWidth / 2)),
+                                    "width": bgWidth,
+                                    "height": bgHeight,
+                                    "stroke": "#aaaaaa",
+                                    "stroke-width": 0.5,
+                                    "class": ("bar-value-text-bg_" + el)
+                                });
+
+                                d3.select(this).style({
+                                    "fill": "white",
+                                    "opacity": "0",
+                                    "fill-opacity": "0",
+                                    "stroke-opacity": 0.75
+                                });
+                            });
+
+                        // add tick mark under shape
+                        parentG.append("rect")
+                            .each(function() {
+                                var rotatedX = parseFloat(b.attr("x")) + parseFloat(barWidth) / 2;
+                                var rotatedY = parseFloat(b.attr("y")) + bgHeight + bgVertOffset - (dialogTickSize / 2);
+                                d3.select(this).attr({
+                                    "transform": (b.attr("transform") + " rotate(45," + rotatedX +"," + rotatedY + ")"),
+                                    "y": (parseFloat(b.attr("y")) + bgHeight + bgVertOffset - dialogTickSize),
+                                    "x": (parseFloat(b.attr("x")) + (parseFloat(barWidth) / 2)),
+                                    "width": dialogTickSize,
+                                    "height": dialogTickSize,
+                                    "class": ("bar-value-text-bg-tick_" + el)
+                                });
+
+                                d3.select(this).style({
+                                    "fill": "white",
+                                    "fill-opacity": "0",
+                                    "opacity": "0"
+                                });
+                            });
+
+                        // add the text value
+                        parentG.append("text")
+                            .each(function() {
+                                d3.select(this).text(function() {
+                                    if (bar.y === 0) {
```

Comment:
Oh, here's where the zero values seem to get filtered out.

Reply:
If these tooltips can't be hidden, I'd rather they say "0.00" than be empty

![image](https://user-images.githubusercontent.com/1430060/30702639-7c16f25a-9ebb-11e7-8e1b-c7be264cabcf.png)